aries-site/src/examples/templates/dashboards/content/chartCards/utils/hooks.js

```diff
@@ -0,0 +1,129 @@
+import { useEffect, useMemo, useState } from 'react';
+import {
+  convertDatesToFeatures,
+  defaultWindow,
+  linearRegression,
+  mean,
+  REPORT_WINDOW_MAP,
+} from '.';
+
+export const useCost = values => {
+  const [meanCost, setMeanCost] = useState(null);
+  const [lastMonthCost, setLastMonthCost] = useState(null);
+  const [projectedCost, setProjectedCost] = useState(null);
+
+  // Calculate cost stats
+  useEffect(() => {
```

Comment:
Any reason that this cannot be executed as a useMemo here?

Reply:
Thanks, have updated to a memo.